runtimes/8.0/Dockerfile

```diff
@@ -40,6 +40,22 @@ RUN apt-get update \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+ARG XDEBUG
+ARG XDEBUG_PORT
+RUN if [ "${XDEBUG}" = 'true' ]; then \
+    apt-get update \
+    && apt-get install -y php-xdebug \
```

Comment:
I had success only with PECL, instead of `apt-get update && apt-get install -y php-xdebug`
```
pecl install xdebug-3.0.3
```

Reply:
Probably because `apt-get install -y php-xdebug` installs a "xdebug.ini" file, so you end up with 2 ".ini", "xdebug.ini" and "php-ext-xdebug.ini".